### ISSUE TITLE
More actions menu is a little out

### ISSUE THREAD

**Issue by robbieaverill** [silverstripe/silverstripe-campaign-admin#119]
![image](https://user-images.githubusercontent.com/5170590/45627019-3e1e5680-ba91-11e8-9819-3a7d9b8e748e.png)

SS 4.3.x-dev

Probably regressed in https://github.com/silverstripe/silverstripe-admin/pull/624 cc @sachajudd

**Comment by robbieaverill** [comment]
There's a missing margin. It's visible when you focus on the more actions menu:

![image](https://user-images.githubusercontent.com/5170590/48346951-71422800-e685-11e8-9404-0670f56da15d.png)

**Comment by sachajudd** [comment]
Yeah I just updated the icon size. I'll take a look at this in the next week.
